analytics.go

```diff
@@ -63,3 +64,29 @@ func (c *Client) GetAggregatedIncidentData(ctx context.Context, analytics Analyt
 	err = c.decodeJSON(resp, &analyticsResponse)
 	return analyticsResponse, err
 }
+
+func (c *Client) GetAggregatedServiceData(ctx context.Context, analytics AnalyticsRequest) (AnalyticsResponse, error) {
```

Comment:
After merging the original PR I raised this one to make some adjustments, could you make your function look more like the style here: https://github.com/PagerDuty/go-pagerduty/pull/310/files -- including the documentation comments?

Reply:
Functions now have the same style as the one you specified, including documentation comments 👍